Sources/Scheduler.swift

```diff
@@ -97,7 +97,16 @@ public final class UIScheduler: SchedulerProtocol {
 	#if os(Linux)
 	private var queueLength: Atomic<Int32> = Atomic(0)
 	#else
-	private var queueLength: Int32 = 0
+	private let queueLength: UnsafeMutablePointer<Int32> = {
```

Comment:
Can you add a comment here explaining why `UnsafeMutablePointer` is used and linking to the thread?